js/src/forum/addPostMentionPreviews.js

```diff
@@ -4,18 +4,19 @@ import PostPreview from 'flarum/components/PostPreview';
 import LoadingIndicator from 'flarum/components/LoadingIndicator';
 
 export default function addPostMentionPreviews() {
-  extend(CommentPost.prototype, 'config', function() {
-    const contentHtml = this.props.post.contentHtml();
+  extend(CommentPost.prototype, 'onupdate', function() {
```

Comment:
The way I see it, this entire block was previously executed on first render, now it won't be. :thinking: